public/js/components/Header/Header.js

```diff
@@ -39,6 +39,13 @@ class Header extends React.Component {
     this.props.atomActions.takeDownAtom(this.props.atom);
   }
 
+  isEditor = () => {
+    return location.pathname.search(new RegExp('\/atoms\/.*\/.*\/edit', 'g')) >= 0;
```

Comment:
This would be nicer if we used react-router's RouteComponent stuff - https://reacttraining.com/react-router/web/example/url-params